examples/unet_inference_3d_array.py

```diff
@@ -75,11 +77,10 @@ def _sliding_window_processor(_engine, batch):
 
 infer_engine = Engine(_sliding_window_processor)
 
-# checkpoint_handler = ModelCheckpoint('./', 'net', n_saved=10, save_interval=3, require_empty=False)
-# infer_engine.add_event_handler(event_name=Events.EPOCH_COMPLETED, handler=checkpoint_handler, to_save={'net': net})
-
-SegmentationSaver(output_path='tempdir', output_ext='.nii.gz', output_postfix='seg').attach(infer_engine)
-CheckpointLoader(load_path='./net_checkpoint_9.pth', load_dict={'net': net}).attach(infer_engine)
+# for the arrary data format, assume the 3rd item of batch data is the meta_data
+SegmentationSaver(output_path='tempdir', output_ext='.nii.gz', output_postfix='seg',
+                  batch_transform=lambda x: x[2]).attach(infer_engine)
+CheckpointLoader(load_path='./runs/net_checkpoint_120.pth', load_dict={'net': net}).attach(infer_engine)
```

Comment:
perhaps add a comment here about how to get `./runs/net_checkpoint_120.pth`?

Reply:
Fixed in the latest patch.
Thanks.